docs/guides/postinstall.md

```diff
@@ -142,6 +142,24 @@ blacklist cdc_mbim" >> /etc/modprobe.d/blacklist.conf'
 
 Please note that this internal ethernet interface is required for various services including touchid that there currently is no Linux support for. In the future, if any of these services are supported, you'll need to undo this.
 
+If this didn't work for you, you can also tell NetworkManager directly to ignore specific interfaces
```

Comment:
We have a udev rule as a better option for this. Will add after this PR. So remove this.